Documentation/using-rkt-with-systemd.md

```diff
@@ -65,6 +65,16 @@ MACHINE CLASS SERVICE
 0 machines listed.
 ```
 
+Note that journald integration is only supported if you have systemd compiled with `xz` compression enabled. One can inspect this using `systemctl`:
```

Comment:
nit, prefer "This can be inspected using `systemctl`" or, "To inspect this, use `systemctl`"